extensions/configuration/servicebindings.yaml

```
@@ -0,0 +1,112 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: servicebindings
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: servicebindings
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |
+    header:
+      - name: spec.controlledBy
+        source: spec.controlledBy
+      - name: status
+        source: 'status.conditions[$count(status.conditions)-1].reason'
+        description: '$filter(status.conditions, function ($v, $i, $a) { $length($v.message) > 0 })[0].message'
+        highlights:
+          informative:
+          - NOTPROVISIONED
+        widget: Badge
+    body:
+      - name: bindingData
+        widget: Panel
+        children:
+          - name: spec.serviceInstanceName
+            source: spec.serviceInstanceName
+          - name: spec.secretName
+            source: spec.secretName
+          - name: spec.externalName
+            source: spec.externalName
+          - name: status.bindingID
+            source: status.bindingID
+          - name: spec.serviceInstanceID
+            source: spec.serviceInstanceID
+      - name: spec.parameters
+        source: spec.parameters
+        widget: CodeViewer
+      - name: spec.parametersFrom
+        source: spec.parametersFrom
+        widget: CodeViewer
+  form: |
+    - simple: true
+      path: spec.serviceInstanceName
+      required: true
+      placeholder: spec.serviceInstanceNamePlaceholder
+    - path: spec.externalName
+      placeholder: spec.externalNamePlaceholder
+    - path: spec.secretName
+    - path: spec.parameters
+      widget: KeyValuePair
+      defaultExpanded: false
+    - path: spec.parametersFrom
+      widget: SimpleList
+      defaultExpanded: true
+      children:
+        - path: '[].secretKeyRef.name'
+          widget: Resource
+          resource:
+            kind: Secret
+            version: v1
+            scope: namespace
+          provideVar: secret
+          required: false
+        - path: '[].secretKeyRef.key'
+          enum: '$keys($secret.data)'
+          required: false
+          placeholder: chooseSecretKey
+  general: |
+    resource:
+      kind: ServiceBinding
+      group: services.cloud.sap.com
+      version: v1
+    name: Service Bindings
+    category: Service Management
+    urlPath: servicebindings
+    scope: namespace
+    description: >-
+      {{[BTP Service Binding](https://github.com/SAP/sap-btp-service-operator#step-2-create-a-service-binding)}}
+      provides access details for an existing service instance.
+  list: |
+    - name: spec.controlledBy
+      source: spec.controlledBy
```

```suggestion
    - name: spec.controlledBy
      source: metadata.ownerReferences
```
Please, adjust translation and name